src/componenrts/Clock/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+interface ClockState {
+  today: Date;
+}
+
+interface ClockProps {
+  clockName: string;
+}
+
+export class Clock extends React.Component<ClockProps, ClockState> {
+  state = {
+    today: new Date(),
+  };
+
```

Comment:
i would have to create constructor, descruct it there and then use this.props.clockName in other functions. I believe this would be worth for bigger projects but in small components like this still worth?

Reply:
Yeah, youre right, leave it like that